### PR TITLE
New Code: Qadir JavaScript Assignment 12 Done.

### PR DESCRIPTION
**Pull request by MKadeer17**

### CODE REVIEW

File: Qadir__Work/JavaScript/Assignment12/Question2/employeeClassData.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    display() {
+        this.calculate();
+        
+        document.getElementById('advanceDetials').innerHTML +=`<br><br>Name: <b>${this.name}</b>`;
+        document.getElementById('advanceDetials').innerHTML +=`<br>Employee No: <b>${this.empno}</b>`;
+        document.getElementById('advanceDetials').innerHTML +=`<br>Annual Gross Salary: <b>${this.gross}</b>`;
+        document.getElementById('advanceDetials').innerHTML +=`<br>Net Salary: <b>${this.net}</b>`;
+    }
+}
+
```
Comment (sanketmod) [inline Qadir__Work/JavaScript/Assignment12/Question2/employeeClassData.js:59]: The solution is incomplete please read the question carefully

**Review by sanketmod — CHANGES_REQUESTED** [review]

File: Qadir__Work/JavaScript/Assignment12/Question2/employeeClassData.js (hunk 1 above)
Comment (MKadeer17) [reply Qadir__Work/JavaScript/Assignment12/Question2/employeeClassData.js:59]: To display the name, empno, gross salary, net salary.
I think I done Properly.
Question is **To display the name, empno, gross salary, net salary.**

Comment (sanketmod) [reply Qadir__Work/JavaScript/Assignment12/Question2/employeeClassData.js:59]: Yes you have to display  the name, empno, gross salary, net salary. but you are displaying annual gross salary not gross salary

**Review by poonam-khode — APPROVED** [review]

File: Qadir__Work/JavaScript/Assignment12/Question2/employeeClassData.js (hunk 1 above)
Comment (MKadeer17) [reply Qadir__Work/JavaScript/Assignment12/Question2/employeeClassData.js:59]: It's Annual gross salary. You check my calculation.

File: Qadir__Work/JavaScript/Assignment12/Question1/employeeClass.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        this.name = "Mohammed Qadir Khan";
+        this.address = "Old Panvel";
+        this.city = "Navi Mumbai";
+        this.salary = 85000;
+        this.da;
+        this.hra;
+        this.pf;
+        this.gross;
+        this.net;
+        this.basic;
```
Comment (Snehal-113) [inline Qadir__Work/JavaScript/Assignment12/Question1/employeeClass.js]: initialize these parameters to 0

File: Qadir__Work/JavaScript/Assignment12/Question2/employeeClassData.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        this.address = address;
+        this.city = city;
+        this.salary = salary;
+        this.da;
+        this.hra;
+        this.pf;
+        this.gross;
+        this.net;
+        this.basic;
+        this.counter = 0;
```
Comment (Snehal-113) [inline Qadir__Work/JavaScript/Assignment12/Question2/employeeClassData.js]: define an empty constructor and use getData method to read values from the user

**Review by Snehal-113 — CHANGES_REQUESTED** [review]

File: Qadir__Work/JavaScript/Assignment12/Question2/employeeClassData.js (hunk 3 above)
Comment (MKadeer17) [reply Qadir__Work/JavaScript/Assignment12/Question2/employeeClassData.js]: ok Mam.

File: Qadir__Work/JavaScript/Assignment12/Question1/employeeClass.js (hunk 2 above)
Comment (MKadeer17) [reply Qadir__Work/JavaScript/Assignment12/Question1/employeeClass.js]: ok Mam.

**Review by Snehal-113 — APPROVED** [review]

**Review by sanketmod — APPROVED** [review]

**Pull request merged**